### ISSUE TITLE
Mail replies to issues not appearing anymore on github issues page

### ISSUE THREAD

**Issue by thierryvolpiatto** [isaacs/github#802]
Hi,
when replying by mail to issues, my replies are not shown on github issues.
Does something have changed ?

BTW let me know if here is the right place to report github issues.

Thanks.

**Comment by NobbZ** [comment]
I observered some delay when replying by mail quite often. The largest delay I observed was about 30 hours.

**Comment by thierryvolpiatto** [comment]
Indeed, some replies are now starting to pop up 48 hours after, but it is recent, I never had this behavior in past years.

Thanks.
